tests/test_openjdk.py

```diff
@@ -212,7 +212,7 @@ def test_jdk_cassandra(container_per_test):
 
     container_per_test.connection.run_expect(
         [0],
-        f"export JAVA_HOME=/usr/ && cd apache-cassandra-{cassandra_version}/ && bin/cassandra -R | tee {logs}",
+        f'export JAVA_HOME=/usr/ && export JVM_OPTS="\$JVM_OPTS -Xss1024k" && cd apache-cassandra-{cassandra_version}/ && bin/cassandra -R 2>&1 >{logs}',
```

Comment:
If the stack size is a problem I would at least double it to avoid future problems of the same kind. The default size is 1024k, and if it blows up on PPC because of a lower value I would immediately set it to 2048k

Reply:
Thanks, done